Editor/ShapeParameters.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Linq;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.ProBuilder;
+
+namespace UnityEditor.ProBuilder
+{
+    internal static class ShapeParameters
+    {
+        static Dictionary<string, Shape> s_Prefs = new Dictionary<string, Shape>();
+
+        static ShapeParameters()
+        {
+            var types = typeof(Shape).Assembly.GetTypes();
```

Comment:
The TypeCache should be used instead: https://docs.unity3d.com/ScriptReference/TypeCache.GetTypesDerivedFrom.html